hack/unwanted-dependencies.json

```diff
@@ -242,6 +247,27 @@
         "google.golang.org/grpc",
         "sigs.k8s.io/apiserver-network-proxy/konnectivity-client"
       ]
-    }
+    },
+    "unwantedVendored": [
+      "cloud.google.com/go/compute",
+      "github.com/GoogleCloudPlatform/k8s-cloud-provider",
+      "github.com/gogo/protobuf",
+      "github.com/golang/mock",
+      "github.com/google/shlex",
+      "github.com/googleapis/enterprise-certificate-proxy",
+      "github.com/googleapis/gax-go/v2",
+      "github.com/gorilla/websocket",
```

Comment:
https://github.com/kubernetes/kubernetes/pull/119157 was merged to remove this vendor. 
```suggestion
```

Reply:
fixed